adagrams/game.py

```diff
@@ -1,13 +1,80 @@
 from random import randint
-
+letters_pool={"A":9,"B":2,"C":2,"D":4,"E":12,"F":2,"G":3,"H":2,"I":9,"J":1,"K":1,"L":4,"M":2,"N":6,"O":8,"P":2,"Q":1,"R":6,"S":4,"T":6,"U":4,"V":2,"W":2,"X":1,"Y":2,"Z":1}
 def draw_letters():
-    pass
+    drawed_letters=[]
+    all_letters=[]
+    letters_pool={"A":9,"B":2,"C":2,"D":4,"E":12,"F":2,"G":3,"H":2,"I":9,"J":1,"K":1,"L":4,"M":2,"N":6,"O":8,"P":2,"Q":1,"R":6,"S":4,"T":6,"U":4,"V":2,"W":2,"X":1,"Y":2,"Z":1}
+    for letter, quantity in letters_pool.items():
+        for i in range(quantity):
+            all_letters.append(letter)
+    while len(drawed_letters)<10:
```

Comment:
Missing whitespace around operator 
```suggestion
    while len(drawed_letters) < 10:
```

Reply:
If I did not know how to play Adagrams, I would not understand the significance of 10. If I were a dev who tried to read this, I might be wondering "Why do we only loop 10 times"?

It would be more descriptive to use a constant variable to reference 10 so that your code is more self-documenting. 

```py
NUM_TILES_ALLOWED_IN_HAND = 10
while len(drawed_letters) < NUM_TILES_ALLOWED_IN_HAND:
```